CartoDB/deep-insights.js
# Disable autostyle if layer visibility is hidden

Fixes #508 

This changes affect the widgets model and view to enable auto-style based on the layer visibility.

### How to test (expected behaviour)
1. Go to the Carto Builder and create a map.
2. Add some widgets based on data from a layer which allow auto-style.
3. Hide the layer from any part of the Builder (Layers list, layer view, etc). The widgets depending on it should not show the auto-style button.
4. Reveal again the layer, and the auto-style buttons should be visible again.

![untitled](https://cloud.githubusercontent.com/assets/2141690/21854422/fbb96cd0-d81a-11e6-8c58-49227f7c0099.gif)
